packages/renderer/src/lib/ingresses-routes/IngressRouteActions.spec.ts

```diff
@@ -0,0 +1,84 @@
+/**********************************************************************
+ * Copyright (C) 2024 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import '@testing-library/jest-dom/vitest';
+import { test, expect, vi, beforeEach, afterEach } from 'vitest';
+import { fireEvent, render, screen } from '@testing-library/svelte';
+import type { IngressUI } from './IngressUI';
+import IngressRouteActions from './IngressRouteActions.svelte';
+import type { RouteUI } from './RouteUI';
+
+const updateMock = vi.fn();
+const deleteIngressMock = vi.fn();
+const deleteRoutesMock = vi.fn();
+
+beforeEach(() => {
+  (window as any).kubernetesDeleteIngress = deleteIngressMock;
+  (window as any).kubernetesDeleteRoute = deleteRoutesMock;
+});
+
+afterEach(() => {
+  vi.resetAllMocks();
+  vi.clearAllMocks();
```

Comment:
reported by @dgolovin one day, reset is doing the clear so we can only do the reset (but I know we have a lot of places with these 2lines, not a requested change)